src/System.Private.ServiceModel/tools/scripts/WCFCustomSetup.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+Echo Set Up test services
+REM
+
+REM ServiceUri is set, then the service is not local hosted
+if NOT "%ServiceUri%"=="" (
```

Comment:
The WCFSetup target has a condition on $(ServiceUri) is empty. So WCFCustomSetup will not get run if it's NOT localhost.
Agree to rename it to be more specific.